src/SmoFacade/Server.cs

```diff
@@ -57,6 +57,34 @@ public void DropLogin(LoginProperties login)
       _server.Logins[login.Name].DropIfExists();
     }
 
+    public decimal DatabaseSizeMb(string dbName)
+    {
+      string query = "SELECT total_size_mb = (SUM(size) * 8. / 1024) " + 
+                     "FROM sys.master_files WITH(NOWAIT) " +

```

Comment:
Nice, this includes log size? That's a good metric to use.

Reply:
correct, the `SUM()` aggregate on the `group by` combines log and data size